test/uk/gov/hmrc/plasticpackagingtax/returns/controllers/returns/ConvertedPackagingCreditControllerSpec.scala

```diff
@@ -75,6 +96,34 @@ class ConvertedPackagingCreditControllerSpec extends ControllerSpec {
 
         status(result) mustBe OK
       }
+
+      "correct date parameters are set in export credits call" in {
+        authorizedUser()
+        mockTaxReturnFind(aTaxReturn())
+        await(controller.displayPage()(getRequest()))
+
+        val dateCaptor = ArgumentCaptor.forClass(classOf[LocalDate])
+        verify(mockExportCreditsConnector).get(anyString(),
+                                               dateCaptor.capture(),
+                                               dateCaptor.capture()
+        )(any())
+        val dates = dateCaptor.getAllValues
+
+        dates.get(0).toString mustBe "2020-04-01"
+        dates.get(1).toString mustBe "2022-03-31"
```

Comment:
It's difficult to see where these dates are coming from. Perhaps you should reference the TaxReturn which is being returned by the mock and show how these are derived from the dates on the attached obligation?

Reply:
I've added some extra asserts just for clarity & a comment to explain.